tests/scripts/create-dev-cluster.sh

```diff
@@ -141,6 +141,13 @@ show_usage() {
 ####################################################################
 ################# MAIN #############################################
 
+
+invocation_error() {
+    echo "$*" >& 2
```

Comment:
How about using `printf` here? It's more versatile and portable.
Like so: `printf "%s\n" "$*" >&2`

Reply:
Yes. I also don't really know how bash interprets `>& 2` instead of `>&2`. This could result in unexpected behavior. I would suggest using `> /dec/stderr` because it is easier to understand for those new to bash.

Reply:
@leseb , @BlaineEXE , thanks for your suggestions. I just implemented them and updated the PR